docs/parsing.md

```diff
@@ -81,6 +81,13 @@ DateTime.fromSeconds(1542674993);
 
 Both methods accept the same options, which allow you to specify a timezone, calendar, and/or numbering system.
 
+### JS Date Object
+
+A native JS `Date` object can be converted into a `DateTime` using [fromJSDate](../class/src/datetime.js~DateTime.html#static-method-fromJSDate).
+
+An optional `zone` option can be provided for accurate DST date computations.
```

Comment:
"Accurate" is not quite the right word, since it implies they'd be inaccurate if the parameter was not provided. "zone-specific" would be better, but IMO even better would be "An optional `zone` parameter can be provided to set the zone on the resulting object", which covers all the bases